scripts/docker_cmake

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
+rm -f  CMakeCache.txt
+rm -rf CMakeFiles/
```

Comment:
My point here is I thought `docker_cmake` as a **configuration** script and, IMO, the cleanup does not belong in it.